services/grafana-loki/0.48.6/defaults/cm.yaml

```diff
@@ -56,6 +56,8 @@ data:
           ingestion_burst_size_mb: 10
           per_stream_rate_limit: 10MB
           per_stream_rate_limit_burst: 15MB
+          # defaults to 30, but ceph components populates 30+ labels that need us to bump this limit.
+          max_label_names_per_series: 40
```

Comment:
should we do it for project-grafana-loki too? doesn't run ceph in there, not sure how many apps populate that many labels

Reply:
since ceph isn't running and we haven't seen any failures i think its okay to not bump there.